src/redux/reducers/create.js

```diff
@@ -413,10 +413,9 @@ export default handleActions(
     },
 
     [actionTypes.CREATE_SET_BOT_BACKGROUND_IMAGE](state, { payload }) {
-      const response = payload;
-      const imagePath = { response };
+      const imagePath = payload.imagePath;
```

Comment:
I would like to point out, `const {imagePath} = payload` was not working, so please ignore this one.